core/tests/Drupal/Nightwatch/Tests/index.js

```diff
@@ -1,9 +1,10 @@
 module.exports = {
-  'Demo Drupal.org': (browser) => {
+  'Test page': (browser, done) => {
     browser
-      .relativeURL('/community')
+      .installDrupal('testing', __dirname + '/index.setup.php')
```

Comment:
Is there any way we can make a guess about what this setup file will be? I think if we can alleviate potential pain points for test writers that will be wonderful.

Reply:
I agree, but well, the function call happens in a different site, so I'm not sure